applications/bci_visualization/operators/color_buffer_passthrough/cpp/color_buffer_passthrough.cpp

```
@@ -0,0 +1,44 @@
+void ColorBufferPassthroughOp::setup(OperatorSpec& spec) {
+  // This drops stale frames if queue is full
+  spec.input<holoscan::gxf::Entity>("color_buffer_in",
+                                   holoscan::IOSpec::kSizeOne,
+                                   holoscan::IOSpec::QueuePolicy::kPop);
+
+  spec.output<holoscan::gxf::Entity>("color_buffer_out");
+}
+
+void ColorBufferPassthroughOp::compute(InputContext& input, OutputContext& output,
+                                       ExecutionContext& context) {
+  auto color_message = input.receive<holoscan::gxf::Entity>("color_buffer_in");
```

Cppcheck (2.19.0)

[error] There is an unknown macro here somewhere. Configuration is required. If HOLOSCAN_OPERATOR_FORWARD_ARGS is a macro then please configure it.

(unknownMacro)